Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM python:3.6-slim
+FROM alpine:3.7
 
-RUN pip install --quiet elasticsearch-curator urllib3[secure]
-ADD ./docker-entrypoint.sh /
-ADD ./config/curator.yml /opt/curator/config/curator.yml
-ADD ./config/action.yml /opt/curator/config/action.yml
+RUN apk add --no-cache python2 py-pip
```

Comment:
why not python 3?

Reply:
Could change that indeed. Used python2 because elastic's curator does so in their own Dockerfile. Had some surprises with python3 and their code.